### PR TITLE
Teams support xsoar8

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9036

## Description
added support to xsoar 8.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:63]:
```suggestion
For Cortex XSOAR version 6.x: `<CORTEX-XSOAR-URL>/instance/execute/<INTEGRATION-INSTANCE-NAME>`, e.g., `https://my.demisto.live/instance/execute/teams`.

For Cortex XSOAR version 8: `https://ext-<CORTEX-XSOAR-Domain>/xsoar/instance/execute/<INTEGRATION-INSTANCE-NAME>`, e.g., `https://ext-my.demisto.live/xsoar/instance/execute/teams`.
```

File: Packs/MicrosoftTeams/ReleaseNotes/1_4_43.md
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/ReleaseNotes/1_4_43.md:6]:
```suggestion
- Added support for Cortex XSOAR 8.
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:69]:
```suggestion
In addition, make sure ***Instance execute external*** is enabled (for Cortex XSOAR 6.x).
```
What is ***Instance execute external**?  Is it a command?  a parameter?  an argument?  I could not find it in the yml file.

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py
Comment (thefrieddan1) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:313]: I would do the server_link assignment in a line before 312
server_link = server_link + '/#' if not is_demisto.... else server_link.
and in line 313 leave it as it was

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py
Comment (thefrieddan1) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:2166]: same here consider setting the server_link with /# in a line before 2166. 
This line as it is now is not very readable

**Review by thefrieddan1 — APPROVED** [review]
Do not merge until there is no approval in slack from @michal-dagan / @yucohen 
Please notice the comments about

**Pull request merged**
